File: package/main/src/Tool/isNotEmpty.ts
/**
 * objectが空かどうかを確認する
 * @param obj
 * @returns {boolean} true: 空でない, false: 空
 */

export const isNotEmpty = (object: object): boolean => {
  // objectが空かどうかを確認する
  return Object.keys(object).length > 0;
};

File: package/umt_python/src/validate/is_not_empty.py
def is_not_empty(obj: object) -> bool:
    """
    Checks if an object is not empty.

    Args:
        obj: The object to check.

    Returns:
        True if the object is not empty, False if it is empty.

    Example:
        >>> is_not_empty({})
        False
        >>> is_not_empty({"a": 1})
        True
    """
    if isinstance(obj, dict):
        return len(obj) > 0
    return False
